src/Registry/Registry.jl

```diff
@@ -451,7 +451,9 @@ function update(regs::Vector{RegistrySpec}; io::IO=stderr_f(), force::Bool=true,
                                     registry_update_log[string(reg.uuid)] = now()
                                     @label done_tarball_read
                                 else
-                                    if reg.name == "General" && Base.get_bool_env("JULIA_PKG_GEN_REG_FMT_CHECK", true)
+                                    if reg.name == "General" &&
+                                            Base.get_bool_env("JULIA_PKG_GEN_REG_FMT_CHECK", true) &&
+                                            get(ENV, "JULIA_PKG_SERVER", "") != ""
```

Comment:
This looks wrong. Not setting `JULIA_PKG_SERVER` is quite different from setting it to an empty string.

Reply:
Should be fixed now?